common/models.py

```
@@ -0,0 +1,77 @@
+class IsDeletedManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(is_deleted=False)
+
+
+class BaseModel(models.Model):
+    """
+    Tracks instance creations, updates, and (soft) deletions.
+    """
+
+    uuid = models.UUIDField(
+        verbose_name=_("UUID"), unique=True, default=uuid.uuid4, editable=False
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)